UMLController.cpp

```diff
@@ -315,3 +331,58 @@ void UMLController::delete_class()
     cout << "Could not find class name " << className << ". Aborting." << std::endl;
   }
 }
+
+
+
+
+
+
+void  UMLController::rename_class()
+{
+  string oldClassName;
+  string newClassName;
+
+  UMLClass nameClass;
+
+  cout << "Enter the CURRENT name of the class you\'d like to rename. -> ";
+  cin >> oldClassName;
+
+  if (!Model.get_class_by_name(oldClassName, nameClass))
```

Comment:
You can simplify this line. The model has a method ` bool does_class_exist(std::string className)` which would make this more readable.